project/profiles/capacity4more/modules/c4m/content/c4m_content_project/c4m_content_project.info

```diff
@@ -45,13 +42,9 @@ features[ds_layout_settings][] = paragraphs_item|c4m_paragraph_text|default
 features[features_api][] = api:2
 features[field_base][] = c4m_cris_contract_number
 features[field_base][] = c4m_cris_decision_number
+features[field_base][] = c4m_implemented_by
```

Comment:
@ordavidil 
We removed it at CFM-1141 (and added 6 others). It's not merged yet,
CFM-1200 derives from develop.